README.md

```diff
@@ -1,4 +1,34 @@
-This repository contains common definitions for the bioconda build system, e.g., the bioconda-utils tag to use and the miniconda version.
+# Overview
 
+This repository acts as a central "source of truth" which can be used by
+various components of the Bioconda build system.
 
-- `common.sh` contains environment variables that control versions
+# Components
+
+**`common.sh`:** contains environment variables that control versions.
+
+**`install-and-set-up-conda.sh`:**
+- downloads and installs conda using the Mambaforge distribution
```

Comment:
We should not use mambaforge anymore. Conda-forge discourages it and advertises miniforge. 

> With the [release](https://github.com/conda-forge/miniforge/releases/tag/23.3.1-0) of Miniforge3-23.3.1-0, that incorporated the changes in https://github.com/conda-forge/miniforge/pull/277, the packages and configuration of Mambaforge and Miniforge3 are now identical. The only difference between the two is the name of the installer and, subsequently, the default installation directory.
Given its wide usage, there are no plans to deprecate Mambaforge. If at some point we decide to deprecate Mambaforge, it will be appropriately announced and communicated with sufficient time in advance.
As of September 2023, the new usage of Mambaforge is thus discouraged. Bug reports specific to Mambaforge will be closed as won't fix.

Reply:
Was not aware of this, thanks for pointing it out.

This PR was mostly about documentation; do you think it's worth changing mambaforge -> miniforge in this PR? *Theoretically* it should not change anything in the build process since we're already on 23.3.1, so this is more a question of "do you think it will *actually* not change anything".

Reply:
A different PR is ok. But we should get rid of it at some point.